src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+<template>
+  <PvPanel class="m-0 p-0 w-full" header="Select params for the Consent and Assent forms">
+    <div class="card flex justify-content-center">
+      <div class="flex flex-wrap gap-3">
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="helpChoose" name="help" value="help" @change="whatDecision" />
+          <label for="helpChoose" class="ml-2">Help me choose</label>
+        </div>
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="iKnow" name="know" value="know" @change="whatDecision" />
+          <label for="iKnow" class="ml-2">I know what to select</label>
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-row">
+      <div class="align-content-center" style="width: 50%" v-if="helpMeSelect">
+        <h3>Default Data Collection</h3>
```

Comment:
Could I suggest that this and `Additional Data Collection` be bolded to match the other header?